models/schedule/kinds/cron.py

```diff
@@ -0,0 +1,22 @@
+from datetime import datetime
+from croniter import croniter
+
+class Cron:
+    def __init__(self, cron):
+        if not croniter.is_valid(cron):
+            raise Exception("Invalid cron")
+
+        self.iter = croniter(cron, datetime.now())
+        self.next_execution = self.iter.get_next(datetime)
+        
+
+    def is_time_to_run(self):
+        current_date_time = datetime.now()
+
+        if (self.next_execution <= current_date_time):
+            self.next_execution = self.iter.get_next(datetime)
+            return True
+
+        current_date_time = datetime.now()
+
+        return False
```

Comment:
```suggestion
    def is_time_to_run(self):
        is_time = self.next_execution <= datetime.now()

        if is_time:
            self.next_execution = self.iter.get_next(datetime)

        return is_time
```

Menos **returns** é mais 😄

Reply:
Considerado no ajuste do factory